__tests__/unit/transactions/transactions.test.ts

```diff
@@ -20,6 +20,7 @@ describe("Transactions", () => {
   const ownerZkpKeys = {};
   const shieldContractAddress = "0x0shieldContractAddress";
   const web3 = {};
+  const tokenId = "";
```

Comment:
No biggie but you can remove this declaration since you are also defining `tokenId` at each transaction `describe` level